### DISCUSSION TITLE
Possible to have Core 2.x.x and Core 3.xx installed at once?

### DISCUSSION THREAD

**Discussion by horendus** [espressif/arduino-esp32#10532] [Q&A]
Is it possible to have core version 2 and 3 installed for use with Arduino IDE 2 with a way to target the version you want to use when compiling?

**Comment by me-no-dev** [comment]
one of them must be installed from git in your Arduino folder. Then you will have both cores accessible through the IDE. The one from git will have `(in sketcbook)` suffix to the name
